bin/prepare_environment.sh

```diff
@@ -31,26 +33,26 @@ done
 
 echo "_______________________CLONING REPOSITORIES_______________________"
 
-git clone -b feat/9-Manage_AI_requests https://github.com/statuscompliance/status-backend.git ../status-backend
+git clone -b feat/9-Manage_AI_requests https://github.com/statuscompliance/status-backend.git status-backend
 
-git clone -b feature/24-Bluejay_integration https://github.com/statuscompliance/status-frontend.git ../status-frontend
+git clone -b feature/24-Bluejay_integration https://github.com/statuscompliance/status-frontend.git status-frontend
 
-git clone https://github.com/statuscompliance/collector-events.git ../collector-events
+git clone https://github.com/statuscompliance/collector-events.git collector-events
 
-cp ../.env.deploy ../status-backend/.env
+cp .env.deploy status-backend/.env
 
-cp ../.env.deploy ../.env
+cp .env.deploy .env
 
 ## If a folder is not created before doing a bind mount in Docker, the folder will be created with root permissions only.
-mkdir -p ../node-red-status
+mkdir -p node-red-status mysql 
```

Comment:
Without creating this folder, further setups of the infrastructure would fail since the files couldn't be removed above (line 27)

Reply:
Thank you for noticing!